app/Models/Course.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Course extends Model
+{
+    public function user()
```

Comment:
xem lại cách đặt tên $this->belongsToMany sẽ trả ra số nhiều thì users phải có `s`